packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_view.dart

```diff
@@ -101,15 +102,15 @@ class _PropertiesListState extends State<_PropertiesList> {
   @override
   void initState() {
     super.initState();
-    // Workaround for https://github.com/flutter/devtools/issues/8929.
+    // Workaround for https://github.com/flutter/flutter/issues/155265.
     setUpTextFieldFocusFixHandler();
```

Comment:
do we still need this since we now have it at the root of DevToolsApp?

Reply:
Good point, removed!